include/date_time.h

```diff
@@ -21,6 +21,30 @@
 extern "C" {
 #endif
 
+/** @brief Date time notification event types used to signal the application. */
+enum date_time_evt_type {
+	/** Date time library has obtained valid time from the modem. */
+	DATE_TIME_FROM_MODEM_OBTAINED,
+	/** Date time library has obtained valid time from NTP servers. */
+	DATE_TIME_FROM_NTP_OBTAINED,
+	/** Date time library has obtained valid time from external source. */
+	DATE_TIME_FROM_EXTERNAL_SOURCE_OBTAINED,
```

Comment:
How about naming these events `DATE_TIME_OBTAINED_*` `MODEM` / `NTP` / `EXT`

Reply:
Yes, this seems cleaner.